### DISCUSSION TITLE
Keycloak purpose

### DISCUSSION THREAD

**Discussion by benjaminbascary97** [ragapp/ragapp#142] [Q&A] [answered]
First, let me congratulate all on the work you been doing lately! With every release, a super useful feature is available and that is super valuable for a SSO project like this, thank you 🫶 

In the other hand... I wanted to know what purpose does cover the Keycloack addition.. does this feature helps adding new users to the /admin routes? New users to the manager? Adding authentication to both /admin and public chat pages? Someone can guide me here?

I've been navigating the new Keycloak layer but Im lost there,

Thank you in advance!

**Comment by marcusschiesser** [comment, marked as answer]
Thanks for your kind words @benjaminbascary97 - always good to get some feedback!

Keycloak is added as a preconfigured auth provider. I just released it; here's what it looks like: https://x.com/MarcusSchiesser/status/1826549718828167197, and here's how to try it yourself: https://github.com/ragapp/ragapp/blob/main/deployments/multiple-ragapps/README.md#how-to-start

**Reply by benjaminbascary97** [reply]
Great, just checking those links right now. I'm just experimenting and testing this in a basic EC2 instance. Thank you!
